### PR TITLE
Distributor contract is now upgradeable

### PR DESCRIPTION
**Pull request by ricardopinto**

### CODE REVIEW

File: contracts/nexus/DistributorFactory.sol [hunk 1]
```diff
@@ -26,7 +26,8 @@ contract DistributorFactory {
     ) public payable returns (address) {
 
         IMemberRoles memberRoles = IMemberRoles(master.getLatestAddress("MR"));
-        Distributor d = new Distributor(
+        Distributor d = new Distributor();
```
Comment (kivanov82) [inline contracts/nexus/DistributorFactory.sol:29]: The problem here is that this DistributorFactory is actually the contract that is managed by Nexus.
So we can't rely on the changes there.
Would it be possible to call 'newDistributor' from our migration and then 'init' on Distributor from our 'owner'?

Comment (ricardopinto) [reply contracts/nexus/DistributorFactory.sol]: Reverted changes on DistributorFactory implementation, and now we might only need to instantiate a Distributor using the `constructor(...)` or `initialize(...)` functions.

**Review by kivanov82 — COMMENTED** [review]
I think there was a misunderstanding of the task.
The 'Distributor' contract should remain as is.
The idea is that we deploy a Proxy, that will _use_ this Distributor, while keeping the 'storage' itself.
So we can completely replace the implementation (witha Distributor v2.0) keeping the same reference to the 'proxy'

**Comment by ricardopinto** [comment]
@kivanov82 the Factory was removed, and Distributer is now instantiated on the migration side as upgradeable

**Pull request merged**